Google.Api.Gax.Grpc/GrpcChannelFactory.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using Grpc.Core;
+
+namespace Google.Api.Gax.Grpc
+{
+    /// <summary>
+    /// Interoperability layer for the aspects of gRPC that aren't covered by Grpc.Core.Api.
+    /// </summary>
+    internal abstract class GrpcChannelFactory
```

Comment:
Left over as well?

Reply:
Whoops, yes. Was considering separating the two, but then didn't.